HED-schema.mediawiki

```diff
@@ -137,16 +148,12 @@ From version 2.2, HED adheres to http://semver.org/ versioning.
 * IAPS <nowiki>[International Affective Picture System]  </nowiki> 
 * IADS <nowiki> [International Affective Digital Sounds]</nowiki> 
 * SAM <nowiki> [The Self-Assessment Manikin]</nowiki> 
+** Music
+*** Tone
 
 '''Sensory presentation''' <nowiki>{extensionAllowed} [Object manifestation]</nowiki> 
 
 * Auditory <nowiki>[Sound]</nowiki> 
-** Nameable 
-** Cash register  
-** Ding <nowiki>[Often associated with positive valence]</nowiki> 
-** Buzz <nowiki>[Often associated with negative valence]</nowiki> 
-** Fire alarm 
-** Click 
 *** ABR <nowiki>[Auditory Brainstem Response]</nowiki> 
```

Comment:
ABR a brain response, not an auditory presentation so I think we should remove this

Reply:
I don't know where to put this so I am leaving temporarily in place for next round and further discussion.